schemas/webpagetest/webpagetest-run/webpagetest-run.1.parquetmr.txt

```diff
@@ -0,0 +1,19 @@
+message webpagetest-run {
+  required binary appName (UTF8);
+  required binary channel (UTF8);
```

Comment:
Would it make sense to include the version and build ID of the application? I'm not sure if we can get this information from WebPageTest, but we at least have the user agent string from the request headers.

Reply:
Absolutely. Version (at least) and buildId (if you can get it) should be there.